src/styles/modules/_m-dropdown.sass

```diff
@@ -23,6 +23,10 @@
       opacity: 1
       visibility: visible
 
+  +is(with-events)
```

Comment:
What does the +is() function do?

Reply:
The reason i'm using the is function here is because it was about the only way I could figure out how to add another class binding when we're using this syntax. :class="b({ searchable: search }).is({ full: full }).is({'with-events': focusedEvents}).toString()"

Reply:
@aap-tim,

Is this the same thing? If so, it does not appear to be supported on Mobile Chrome or Samsung. 
https://developer.mozilla.org/en-US/docs/Web/CSS/:is#Browser_compatibility

Reply:
@shannon-aap 

That does not look like the same thing. I believe it's just a function Kevin created to tack is on to the beginning of a CSS class. It looks like people use is as a way to describe different states of a block or element or component in BEM. https://www.smashingmagazine.com/2016/06/battling-bem-extended-edition-common-problems-and-how-to-avoid-them/

Reply:
I ended up using it here because the way we're using the b function in our vue templates makes it un-intuitive for me to add on an extra class binding the way we normally would with vue. For example: "{isActive: active, isFull: full}". I'm not sure how to do that with this syntax.

<img width="364" alt="Screen Shot 2019-12-11 at 11 31 00 AM" src="https://user-images.githubusercontent.com/46794759/70640342-d0b49380-1c09-11ea-81df-d33f8a1ea83b.png">

Reply:
Thanks for the explanation, @aap-tim